### PR TITLE
Bugfix/toastr css updates

### LINKED ISSUE #248: New toastrs need style adjustments for dark mode
Toastr styles aren't working well with bs4 from ngx-toastr

### PR DESCRIPTION
**Pull request by therobbiedavis**
Closes #248

### CODE REVIEW

**Comment by majora2007** [comment]
# Issues
- I'm seeing the background image flash on series-detail page, despite it not supposed to be showing there.
- The background image is too large, even after minification, it's sitting at 2MB which is causing a slow first paint and a flash on loading of login page. 
- Why are we adding a new font family?

File: src/app/user-login/user-login.component.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    align-items: center;
+    justify-content: center;
+    height: calc(100vh);
+    min-height: 289px;
+    position: relative;
+
+
+    &::before {
+        content: "";
+        background-image: url('../../assets/images/alfons-morales-YLSwjSy7stw-unsplash-min.jpg');
```
Comment (majora2007) [inline src/app/user-login/user-login.component.scss]: Rename this file to be something more meaningful at first glance

File: src/app/user-login/user-login.component.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            border-color: #444f75;
+        }
+
+        .alt:hover {
+            background-color: #3b4466;
+        }
+
+        .alt:focus {
+            background-color: #343c59;
+            box-shadow: 0 0 0 0.2rem rgb(68 79 117 / 50%)
```
Comment (majora2007) [inline src/app/user-login/user-login.component.scss]: Please keep semicolons after all CSS styles.

File: src/styles.scss [hunk 3]
```diff
@@ -1,12 +1,13 @@
-// Import colors for overrides of bootstrap theme
-@import './theme/colors';
-
 @import '~bootstrap/scss/bootstrap';
 @import '~bootstrap/scss/mixins/_breakpoints.scss';
 
-@import '~ngx-toastr/toastr-bs4-alert';
+//@import '~ngx-toastr/toastr-bs4-alert';
```
Comment (majora2007) [inline src/styles.scss]: Remove this dead code

**Comment by therobbiedavis** [comment]
- Flashing should be fixed.
- I can resize the background image to help the size
- New font family is for "Kavita" branding

**Review by majora2007 — CHANGES_REQUESTED** [review]

**Review by majora2007 — APPROVED** [review]

**Pull request merged**